.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "gradle"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
```suggestion
```

How about let's just leave it to checking the github actions versions, that seems pretty nice actually.

Reply:
> How about let's just leave it to checking the github actions versions, that seems pretty nice actually.

Done.